configure.ac

```diff
@@ -585,6 +585,36 @@
         AC_MSG_RESULT(no)
     fi
 
+  # libhs
```

Comment:
I noticed that hyperscan ships with a pkg-config file. I think it would make sense to use that here?